pkgs/applications/networking/browsers/webmacs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, qtbase
+, herbstluftwm
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "webmacs";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner  = "parkouss";
+    repo   = "webmacs";
+    rev    = "d692ae8c2bf82cce854f34cb3d762932fe7fdc8b";
+    fetchSubmodules = true;
+    sha256 = "1hzb9341hybgrqcy1w20hshm6xaiby4wbjpjkigf4zq389407368";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    python3Packages.pyqt5_with_qtwebkit
```

Comment:
```suggestion
    pyqt5_with_qtwebkit
```

Reply:
with python3Packages brings it into scope for the whole list, and so you do not need to repeat it inside the list

Reply:
Yes, that's absolutely clear, I just forgot to remove the qualifications after adding the `with`.